src/main/app/src/app/core/service/util.service.ts

```diff
@@ -1,36 +1,21 @@
 /*
- * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-FileCopyrightText: 2024 Lifely
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
```

Reply:
FYI; the ProgressDialogComponent is newly created and so fully 2024 Lifely